Arrays.java

```diff
@@ -0,0 +1,24 @@
+public class Arrays {
+
+    public static void main(String[] args) {
+        int[] numbers = new int[56];
+        int average = 0;
+        int sum = 0;
+        for (int i = 1; i < numbers.length; i++) {
+            numbers[i] = i;
+            sum += numbers[i];
+            average = sum / numbers.length;
```

Comment:
Let's move counting of average out from cycle, we need make this action once after sum has already been prepared